index.html

```diff
@@ -265,9 +265,9 @@ <h1 class="title">Contact me</h1>
             to help with it
           </p>
 
-          <form action="" class="contact-form" get>
+          <form action="https://formspree.io/f/mzbwrolk" class="contact-form" method="post">
             <input type="text" placeholder="Alexandra Loremipsum" />
-            <input type="email" placeholder="aleandra@gamil.com" />
+            <input type="email" placeholder="aleandra@gamil.com" name="email" id="email"/>
             <textarea
               name="textarea"
               id="comment"
```

Comment:
### Your Form
![image](https://user-images.githubusercontent.com/93039359/195415747-5103f7b9-d7c8-4b91-a877-2fbff4e47d52.png)

### Figma Form
![image](https://user-images.githubusercontent.com/93039359/195415974-11aa6388-983e-456f-af25-e62ae5d8046c.png)

**[Optional]**

- [ ] Your design is not much similar to the Figma template :thinking: Kindly make your design look similar to the Figma template to make it professional :blush: